index.js

```diff
@@ -1,9 +1,10 @@
 module.exports = robot => {
   robot.on('issues.opened', async context => {
-    const options = context.repo({path: '.github/ISSUE_REPLY_TEMPLATE.md'});
-    const res = await context.github.repos.getContent(options);
-    const template = new Buffer(res.data.content, 'base64').toString();
-
-    return context.github.issues.createComment(context.issue({body: template}));
+    const config = await context.config('ISSUE_REPLY_TEMPLATE.yml');
```

Comment:
To be completely honest, I had no idea people were even using this. My thinking was along the lines of "as an example bot, what examples should we be showing?"

> Here's my current thinking on config patterns... For plugins that predominantly have templated content, I think I prefer using a .md... For plugins that have multiple config values, I prefer the a YAML file

I definitely agree that this makes sense. Having never used the template files myself, it's easy to forget they exist, but that consistency is definitely nice. But you're totally right, we shouldn't break that existing functionality. 

>  We should have a more in-depth chat soon about config patterns and document best practices.

I totally agree, it would be great to document this thoroughly.

I've also been doing some thinking on how to get bots working on an organizational level and whether that should be something probot could support (ie reading from a config file in one central (maybe docs or config ?) repo and using those values for every repo in the org). I was actually considering make a bot framework-ish thing to help do that. Because I feel like I frequently have to make changes to all the behaviorbot plugins in the org and it can be annoying going through each one and, for example, adding the same config file to each.